### PR TITLE
replaced err = {} in catch

### PR DESCRIPTION
**Pull request by SpeckyYT**
I caused this "Unexpected token =" error, sorry

### CODE REVIEW

File: src/Player.js [hunk 1]
```diff
@@ -168,8 +168,8 @@ class Player extends EventEmitter {
 
             return song;
         }
-        catch (err = {}) {
-            this.emit('error', err.message || err, message);
+        catch (err) {
+            this.emit('error', err ? err.message : err, message);
```
Comment (SushiBtw) [inline src/Player.js:172]: We should use `err.message ?? err`, as if err is a string it will still log `undefined`.

File: src/Player.js [hunk 2]
```diff
@@ -274,8 +274,8 @@ class Player extends EventEmitter {
 
             return playlist;
         }
-        catch (err = {}) {
-            this.emit('error', err.message || err, message);
+        catch (err) {
+            this.emit('error', err ? err.message : err, message);
```
Comment (SushiBtw) [inline src/Player.js:278]: We should use `err.message ?? err`, as if err is a string it will still log `undefined`.

**Review by SushiBtw — CHANGES_REQUESTED** [review]
pls pls

**Comment by SpeckyYT** [comment]
fixed

**Review by SushiBtw — APPROVED** [review]

**Pull request merged**